tests/contrib/test_base_models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import connection, models
+from django.db.models import base
+from django.core.management.color import no_style
+from django.utils import timezone
+
+import freezegun
+import pytest
+
+from euth.contrib import base_models
+
+@pytest.fixture
+def time_stamped_model(request, db):
```

Comment:
I would suggest to put fixtures into ` tests/contrib/conftest.py` so that they are separated from the tests and can be reused by other tests.